### PR TITLE
Add `/workspaces/create`

### PR DESCRIPTION
**Pull request by andrii-balitskyi**

### CODE REVIEW

File: test/api/internal/workspaces/create.test.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import test, { type ExecutionContext } from "ava"
+
+import { getTestServer } from "fixtures/get-test-server.ts"
+
+test("GET /internal/workspaces/create", async (t: ExecutionContext) => {
```
Comment (phpnode) [inline test/api/internal/workspaces/create.test.ts:5]:
```suggestion
test("POST /internal/workspaces/create", async (t: ExecutionContext) => {
```

File: test/api/workspaces/create.test.ts [hunk 2]
```diff
@@ -1,38 +1,24 @@
 import test, { type ExecutionContext } from "ava"
 
 import { getTestServer } from "fixtures/get-test-server.ts"
+import { seed } from "lib/database/seed.ts"
 
 test("GET /workspaces/create", async (t: ExecutionContext) => {
```
Comment (phpnode) [inline test/api/workspaces/create.test.ts:6]:
```suggestion
test("POST /workspaces/create", async (t: ExecutionContext) => {
```

**Review by phpnode — APPROVED** [review]

File: test/api/workspaces/create.test.ts (hunk 2 above)
Comment (andrii-balitskyi) [reply test/api/workspaces/create.test.ts]: https://github.com/seamapi/fake-seam-connect/pull/189/commits/c490e03ca6ce172e239842a1aa21d30a6589268f

File: test/api/internal/workspaces/create.test.ts (hunk 1 above)
Comment (andrii-balitskyi) [reply test/api/internal/workspaces/create.test.ts]: https://github.com/seamapi/fake-seam-connect/pull/189/commits/c490e03ca6ce172e239842a1aa21d30a6589268f

**Pull request merged**